### PR TITLE
MM Rebranding

### PR DESCRIPTION
**Pull request by tomasMizera**
PR replaces logos with new ones for the rebranding.

An additional icon is added at the beginning of the MM toolbar with the MM company icon - on click it opens the MM webpage.

I am wondering if we should not do the transition of toolbar icons in two phases so that users have a chance to get used to the new logo. I find the new toolbar visually way too different to do it in one go. Previous: 
image

Options for the new one:
 - a) image new logo + legacy icons
 - b) image new logo + legacy icons (but without legacy mergin logo) ~ we can make these icons smaller and have border just like on the previous option
 - c) image new logo + new icons from new icon pack

Which one would you prefer @wonder-sk / @PeterPetrik ?

---

There is also a new `QMessageBar` message informing about rebranding - this message is shown to authorised users. It is visible only once.

---

**Bonus question for @wonder-sk**: We also change the plugin's metadata - **name**, is that ok? Is it still going to be the same plugin in the QGIS plugin repository?

---

_Some screenshots:_

image

image

image

image

### CODE REVIEW

**Comment by alexbruy** [comment]
> We also change the plugin's metadata - name, is that ok? Is it still going to be the same plugin in the QGIS plugin repository?

If I'm not wrong, if we change name it will be new plugin from the plugin repository point of view.

**Comment by alexbruy** [comment]
@tomasMizera shouldn't we change dialog titles too? E.g. "Mergin Settings" to "MerginMaps Settings" and so on.

**Review by alexbruy — APPROVED** [review]
Code looks good to me.

File: Mergin/metadata.txt [hunk 1]
```diff
@@ -1,6 +1,6 @@
 ; the next section is mandatory
 [general]
-name=Mergin
+name=Mergin Maps
```
Comment (PeterPetrik) [inline Mergin/metadata.txt:3]: keep the old name please

**Review by PeterPetrik — APPROVED** [review]

**Comment by PeterPetrik** [comment]
very nice. 

I suggest to keep old name ("Mergin") for plugin name s it is same in the qgis plugin repo.

- for icons, I am easy with both a) -> c) (in phase 3) or immediately c).  I will leave it with you. I will not go with b) 
- also for notification, we can remove it in phase 3

**Comment by tomasMizera** [comment]
Allright, let's go with the option a) in this phase.

**Pull request merged**